### PR TITLE
feat: Add OWNERS_ALIASES and edit OWNERS file

### PR DESCRIPTION
**Pull request by opokornyy**
**What this PR does / why we need it**:

This PR should automate  adding sradco to all monitoring PRs

**Release note**:
```
None
```

### CODE REVIEW

File: OWNERS
Comment (0xFelix) [inline OWNERS:3]: You could add `docs/metrics.dm` and `‎tests/monitoring_test.go` too.

File: OWNERS_ALIASES
Comment (0xFelix) [inline OWNERS_ALIASES:9]: Add newline between groups?

File: OWNERS_ALIASES
Comment (lyarwood) [inline OWNERS_ALIASES:23]: @sradco should others also be listed here?

**Review by lyarwood — COMMENTED** [review]
This `/lgtm` but I wonder if there's a way of testing this somewhere before we merge?

File: OWNERS_ALIASES
Comment (opokornyy) [reply OWNERS_ALIASES:23]: I have asked @sradco yesterday and there is no need to add someone else now.

**Review by 0xFelix — APPROVED** [review]
Thanks!

/approve

**Pull request merged**
